randovania/games/super_metroid/json_data/Crateria.txt

```diff
@@ -446,21 +461,31 @@ Extra - asset_id: 13
 > Door to Terminator Room; Heals? False
   * Layers: default
   * Normal Door to Terminator Room/Door to Green Pirates Shaft
-  > Door to Lower Mushrooms
-      Trivial
   > Door to Gold Four Hallway
-      Trivial
+      Any of the following:
+          Charge Beam or Ice Beam or Missile or Plasma Beam or Power Bombs or Spazer Beam or Super Missile or Wave Beam or Can Lay Any Bombs
```

Comment:
This is still mising a comment. something like "defeat pirate" or something.

Reply:
Added note to defeat green pirates